recipes/keras2pmml/meta.yaml

```diff
@@ -0,0 +1,37 @@
+{% set name = "keras2pmml" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2af5a5a5824b0bdcb864eb83a7e7dd2d7a6ae1cad73c7b8f5dc4d1b6f06db3df
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - setuptools
+  run:
+    - numpy >=1.6.1
+    - scipy >=0.9
+    - theano >=0.8.2
+    - keras >=1.0.6
+    - scikit-learn >=0.17.1
+
+test:
+  imports:
+    - "{{ name }}"
```

Comment:
would it be possible to have more import tests? Also can you please hard code the import test?